extensions/rescuetime-focus-session-trigger/src/startFocusTime.ts

```diff
@@ -0,0 +1,44 @@
+import { getPreferenceValues, showToast, Toast } from "@raycast/api";
+import got from "got";
+
+export default async () => {
+  const toast = await showToast({
+    style: Toast.Style.Animated,
+    title: "Starting Focus Session",
+    message: "RescueTime desktop app can take up to 1 minute to reflect Focus Session",
+  });
+
+  interface Preferences {
+    APIkey: string;
+    duration: string;
+  }
+
+  const preferences = getPreferenceValues<Preferences>();
+
+  if (isNaN(parseInt(preferences.duration, 10))) {
+    showToast({
+      style: Toast.Style.Failure,
+      title: "Default Focus Session Durration Preferences needs to be a number",
```

Comment:
```suggestion
      title: "Failed to start Focus Session",
      message: "Default focus session duration needs to be a number",
```
Could we rephrase it like this? Corrected a spelling error and added the text as a message